src/System.Net.NameResolution/tests/PalTests/NameResolutionPalTests.cs

```diff
@@ -105,7 +105,7 @@ public void TryGetAddrInfo_HostName_TryGetNameInfo()
             {
                 // On Unix, getaddrinfo returns host not found, if all the machine discovery settings on the local network
                 // is turned off. Hence dns lookup for it's own hostname fails.
-                Assert.True(RuntimeInformation.IsOSPlatform(OSPlatform.Linux) || RuntimeInformation.IsOSPlatform(OSPlatform.OSX));
+                Assert.True(RuntimeInformation.IsOSPlatform(OSPlatform.Linux) || RuntimeInformation.IsOSPlatform(OSPlatform.OSX) || RuntimeInformation.IsOSPlatform(OSPlatform.FreeBSD));
```

Comment:
Seems like these should be simplified to either:
```C#
Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Windows));
```
or
```C#
Assert.Equal(PlatformID.Unix, Environment.OSVersion.Platform);
```
since we're really just checking to see if this is Unix / not Windows.